SwissTransferFeatures/NewTransferView/NewTransferSettingsView.swift

```diff
@@ -0,0 +1,166 @@
+/*
+ Infomaniak SwissTransfer - iOS App
+ Copyright (C) 2024 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import InfomaniakCoreUI
+import STResources
+import SwiftUI
+import SwissTransferCore
+import SwissTransferCoreUI
+
+
+struct NewTransferSettingsView: View {
+    @State private var duration = 30
+    let durations: [Int] = [30, 15, 7, 1]
+
+    @State private var limit = 250
+    let limits: [Int] = [250, 100, 20, 1]
+
+    @State private var showPasswordSetting = false
+
+    @State private var language: TransferLanguage = .fr
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: IKPadding.medium) {
+            Text(STResourcesStrings.Localizable.advancedSettingsTitle)
+                .font(.ST.callout)
+                .foregroundStyle(Color.ST.textPrimary)
+
+            VStack(alignment: .leading, spacing: 32) {
+                Menu {
+                    ForEach(durations, id: \.self) { value in
+                        Button(STResourcesStrings.Localizable.settingsValidityPeriodValue(value)) {
+                            duration = value
+                        }
+                    }
+                } label: {
+                    HStack {
+                        Label {
+                            Text(STResourcesStrings.Localizable.settingsOptionValidityPeriod)
+                                .font(.ST.calloutMedium)
+                                .foregroundStyle(Color.ST.primary)
+                        } icon: {
+                            STResourcesAsset.Images.clock.swiftUIImage
+                                .resizable()
+                                .frame(width: 16, height: 16)
+                        }
+                        .labelStyle(.horizontal)
```

Comment:
create a custom label style for the app which adds an 8pt space between the icon and the text

Reply:
Not needed. Label already have a 8 spacing